### ISSUE TITLE
Laravel 7 & 8 show "Unknown" for most transactions

### ISSUE THREAD

**Issue by swichers** [newrelic/newrelic-php-agent#589]
After updating to the latest agent on our servers, transactions started being reported as "unknown" for most requests. We are currently running v10.3.0.315 on PHP 7.4.33. This happens with Laravel versions:

* v7.30.6
* v8.83.25
* v8.83.26

It is unrelated to any code deployments; the problem started after server updates. I do not have the versions that were running prior to the updates. This impacts multiple applications and multiple servers. Drupal applications remain unaffected.

![image](https://user-images.githubusercontent.com/5890607/205751780-d9d8f6af-2aa7-4d8b-9f3e-912636f15efb.png)

Pink here represents when "unknown" started to dominate our transactions.

As part of troubleshooting we completely removed New Relic, all custom configuration, all agents, and then did a fresh install of everything using defaults. This had no change in behavior. Also tried forcing an opcache clear on one server (based on #387) with no change.

It's worth pointing out that while do use nginx+PHP7.4, other PHP versions are available on the server.

I have a post on the community forums that has not seen any useful progress. I am hoping for some direction on this issue. Are there relevant logs that would be useful? Some configuration to check? Some symlink to verify? New Relic has been effectively useless to us for 3 weeks at this point.

This is similar to #387 but obviously we are on PHP 7, not 8/8.1, and the latest version of NR+agent did not resolve this issue.

**Comment by bduranleau-nr** [comment]
Hi @swichers, thanks for bringing this issue to our attention.

For a bit more background information to help us diagnose, are you able to confirm that this issue was *NOT* occurring prior to upgrading to agent version 10.3.0.315? Also, are you able to share the list of extensions your PHP app is running so we can rule out interference from other PHP extensions?

For clarification and to make sure we're on the same page:

> the problem started after server updates.

Were the server updates in this case solely upgrading the agent, or were other changes implemented as well? If there are other changes that were implemented, are you able to share them here?

> I do not have the versions that were running prior to the updates.

Is this referring to the New Relic Agent version, or something else? If not the agent, can you share what agent version you upgraded from?

**Comment by swichers** [comment]
@bduranleau-nr This was _not_ happening until updating New Relic to the latest versions. Basically everything was fine, we updated PHP and New Relic, and now it says "unknown" for most Laravel transactions.

"Server update" in this case was PHP + New Relic agent + PHP agent.

PHP was updated from 7.4.32 to 7.4.33. I do not have concrete information about the previous NR versions, but they were possibly 1.27.4 for the main agent and possibly v9.21.0.311 for the PHP agent. This is based on information being reported in New Relic after the updated happened, but before the environment section updated to show the new versions.

Module list:

```
bcmath
calendar
cgi-fcgi
ctype
curl
date
dom
exif
FFI
fileinfo
filter
ftp
gd
gettext
gmp
hash
http
iconv
igbinary
imap
intl
json
libxml
mbstring
memcached
msgpack
mysqli
mysqlnd
newrelic
openssl
pcre
PDO
pdo_mysql
pdo_sqlite
Phar
posix
propro
raphf
readline
redis
Reflection
session
shmop
SimpleXML
soap
sockets
sodium
SPL
sqlite3
standard
sysvmsg
sysvsem
sysvshm
tokenizer
xml
xmlreader
xmlrpc
xmlwriter
xsl
Zend OPcache
zip
zlib
```

**Comment by bduranleau-nr** [comment]
Thank you for confirming all of this @swichers.

One suggestion we can make at the moment is to try to downgrade your agent to [10.2.0.314](https://docs.newrelic.com/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-2-0-314/) and see if the issue persists.

In order to get more information to diagnose this issue that may be too sensitive to share on github (such as logs and other private details), we suggest filing a ticket by following the steps [here](https://docs.newrelic.com/docs/new-relic-solutions/solve-common-issues/find-help-use-support-portal/#file-ticket), where we can engage our General Tech Support engineers.

If you do not have access to this support flow & the ability to open a support ticket directly, we can reach out via a post on the NR [Community Forums](https://discuss.newrelic.com/). You mention in your initial post that you made a post already, can you share a link to that here so I can pass it along to our GTS engineers? (if you are able to create a support ticket directly, you can ignore this step)

**Comment by swichers** [comment]
@bduranleau-nr I appreciate your response here and suggestion on what to try. On one server we have downgraded to 10.2.0.314 for the following:

- newrelic-php5
- newrelic-php5-common
- newrelic-daemon

We have also done a restart of nginx after this downgrade. I will monitor to see if requests from that server still show "unknown" and update here.

We are on a PAYG plan and do not have access to actual support tickets. This is the community post I made https://discuss.newrelic.com/t/php-laravel-routes-show-as-unknown-after-agent-update/191444

**Comment by swichers** [comment]
That downgrade looks promising. No "unknowns" on that server in the last 3 hours.

**Comment by swichers** [comment]
@bduranleau-nr Downgraded all servers today and "unknowns" no longer appear. Interestingly enough, overall application performance improved with the downgrade of New Relic

![image](https://user-images.githubusercontent.com/5890607/206623014-72520ab8-863a-4052-a60e-ab7c693d7f07.png)

**Comment by pgibson1-godaddy** [comment]
@swichers performance improvement is likely due to https://github.com/newrelic/newrelic-php-agent/issues/577

**Comment by swichers** [comment]
Issue has been resolved in the beta 10.4.0.364, and the latest 10.5 is supposed to have the same fixes.
